lm75bd/lm75bd.c

```diff
@@ -9,6 +9,7 @@
 
 /* LM75BD Registers (p.8) */
 #define LM75BD_REG_CONF 0x01U  /* Configuration Register (R/W) */
+#define LM75BD_OBC_I2C_ADDR 0b1001111U /* Peripheral Address*/
```

Comment:
This only needs to be defined in 1 place so remove this. It will be used from the header file